src/index.ts

```diff
@@ -74,6 +71,10 @@ interface ActualDefinitionNode {
 const undefinedVariableConst = "undefined_variable"
 const isVariableDropinConst = "_____isVariableDropinConst"
 
+const isString = (arg: any): boolean => typeof arg === "string"
+const isArray = Array.isArray
+const isObject = (arg: any): boolean => arg instanceof Object
```

Comment:
Do you know if `instanceof` also works in older nodejs versions @thomaschaplin ?

Reply:
Ah, seems like this exists since node 0.10.0 - perfect :)